include/eve/detail/function/simd/common/broadcast_group.hpp

```diff
@@ -0,0 +1,101 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/abi.hpp>
+#include <eve/pattern.hpp>
+
+namespace eve::detail
+{
+  template<simd_value Wide, std::ptrdiff_t Group, std::ptrdiff_t Index, std::ptrdiff_t Size>
+  requires(   (Group >  0) && (Group <= std::min(cardinal_v<Wide>, Size))
+          &&  (Index >= 0) && (Index < cardinal_v<Wide>/Group)
+          )
+  EVE_FORCEINLINE auto broadcast_group_ ( EVE_SUPPORTS(cpu_), Wide w
+                                        , fixed<Group> g, index_t<Index> i, fixed<Size> sz
+                                        )
```

Comment:
I am a bit lost.

I understand what index and size is.
Like second pair of elements.
What's Group?

Reply:
The size of the group you broadcast.
You broadcast the Index-th group of Group values into a Size wide as output.